### PR TITLE
qbs: add head spec

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/qbs.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class Qbs < Formula
   url "https://download.qt.io/official_releases/qbs/1.12.0/qbs-src-1.12.0.tar.gz"
   sha256 "5efeb2492f8ccf0e7a5ea106f748e1c536f964674025aecb22c1ee948e3e35d1"
 
+  head "https://code.qt.io/qbs/qbs.git"
```
Comment (commitay) [inline Formula/qbs.rb]: No newline between `sha256` and `head` please.

**Comment by yurikoles** [comment]
BTW, qbs is marked as [deprecated](https://blog.qt.io/blog/2018/10/29/deprecation-of-qbs/). Should we add some caveat for this statement?

**Comment by yurikoles** [comment]
@commitay bump?

**Comment by fxcoudert** [comment]
> Qbs will remain supported until the end of 2019 with the last planned release in April 2019, together with Qt Creator 4.9

**Pull request merged**